lib/api_helpers.rb

```diff
@@ -0,0 +1,101 @@
+##
+# helper methods for handing external API requests/responses
+##
+module ApiHelpers
+  # check if OK response code was found
+  #
+  # * *params*
+  #   - +code+ (Integer) => integer HTTP response code
+  #
+  # * *return*
+  #   - +Boolean+ of whether or not response is a known 'OK' response
+  def ok?(code)
```

Comment:
that's really weird that there's not a built-in Rails method for this, because it seems so useful, but a few minutes of googling didn't turn up anything.